umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -181,12 +198,12 @@ Ensure you have node-red-contrib-kafkajs installed. If not, see
 
 Add a kafka-producer node, connecting it to the JSON node. Configure as follows:
 
-1. Open the configuration menu by double-click on the kafka-producer node.
-After that, click on the edit button.
+1.  Open the configuration menu by double-click on the kafka-producer node.
+    After that, click on the edit button.
 
-    ![Node-RED Kafka Producer](/images/getstarted/dataAcquisitionManipulation/noderedKafkaProducer.png)
+        ![Node-RED Kafka Producer](/images/getstarted/dataAcquisitionManipulation/noderedKafkaProducer.png)
```

Comment:
This image is not displayed correctly

Reply:
Not sure what you mean, this image was already there and seems to show up fine for me.

Reply:
<img width="511" alt="image" src="https://github.com/united-manufacturing-hub/umh.docs.umh.app/assets/129943599/1d95245f-c1e1-4cdd-8d0c-d115238e6d4c">

looks like this when I open the preview.